src-11ty/_includes/base.html

```diff
@@ -0,0 +1,192 @@
+{% capture full_title -%} {%- if title -%} {{ title }} – Device Toolbox {%- else
+-%} Device Toolbox {%- endif -%} {%- endcapture %}
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>{{ full_title }}</title>
+    <link rel="icon" type="image/png" href="/favicon.png" />
+    <meta name="description" content="{{ description }}" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://early.webawesome.com/webawesome@3.0.0-beta.6/dist/styles/webawesome.css"
+    />
+    <script type="module" src="/src/utils/dark-mode.ts"></script>
+
+    <link rel="stylesheet" href="/src/styles/shoelace.css" />
+    <link rel="stylesheet" href="/src/styles/theme.css" />
+    <link rel="stylesheet" href="/src/styles/details-page.css" />
+    <link
+      rel="preload"
+      href="/open-home-foundation.svg"
+      as="image"
+      type="image/svg+xml"
+    />
+    <link rel="preload" href="/ohf_dark.svg" as="image" type="image/svg+xml" />
+    <style>
```

Comment:
For a future PR, we might want to consider extracting this into a style sheet, or maybe even make a master stylesheet that inlines shoelace, theme, details-page and base CSS